### PR TITLE
Fix `SorterAnalyzer` docstrings

### PR DESCRIPTION
**Pull request by zm711**
I figure it is better just to add some updates live so that it will be easier to do small chunks rather than giant edits of the docstrings. But if you want me to close this and wait until later to do this that's fine too. Also (pre-weekend @samuelgarcia :).

### CODE REVIEW

File: src/spikeinterface/core/sortinganalyzer.py [hunk 1]
```diff
@@ -141,17 +141,17 @@ class SortingAnalyzer:
...
-    This internaly maintain a list of computed ResultExtention (waveform, pca, unit position, spike poisition, ...).
+    This internally maintains a list of computed ResultExtention (waveform, pca, unit position, spike position, ...).
 
     This can live in memory and/or can be be persistent to disk in 2 internal formats (folder/json/npz or zarr).
     A SortingAnalyzer can be transfer to another format using `save_as()`
 
     This handle unit sparsity that can be propagated to ResultExtention.
 
-    This handle spike sampling that can be propagated to ResultExtention : work only on a subset of spikes.
+    This handle spike sampling that can be propagated to ResultExtention : works at only on a subset of spikes.
```
Comment (zm711) [inline src/spikeinterface/core/sortinganalyzer.py:151]:
```suggestion
    This handle spike sampling that can be propagated to ResultExtention : works on only a subset of spikes.
```

**Comment by samuelgarcia** [comment]
Hi Zach.
Feel free to make some PR when you have time.
I merge then immediatly to avoid collision in dev branch.
You can keep the same branch for this and re open PR as soon you want to fight with my typos.
Note that I did not yet have time myself to read docstring again so the typos rate should be very high!
Thanks a lot for this help.

**Comment by zm711** [comment]
Sounds good Sam. I'll try to test out the SortingAnalyzer next week on some of my data.

**Comment by samuelgarcia** [comment]
cool thanks.
spikeinterface-gui has also a dev branch which should work on it.

**Pull request merged**